qa/rpc-tests/p2p-versionbits-warning.py

```diff
@@ -74,7 +74,7 @@ def setup_network(self):
         self.nodes.append(start_node(0, self.options.tmpdir, self.node_options))
 
         import re
```

Comment:
Yeah, unless there is a specific reason for late binding (there doesn't seem to be here)